presets/ragengine/main.py

```diff
@@ -177,9 +245,19 @@ async def query_index(request: QueryRequest):
     """,
 )
 def list_indexes():
```

Comment:
I realize I have a bug here - can you change this to async?

Reply:
@ishaansehgal99 Is this a necessary change? I don't see any I/O here

Reply:
ok lets leave it

```
    def list_indexes(self) -> List[str]:
        return list(self.index_map.keys())
```

is sync